### PR TITLE
Take exports seriously

### PR DESCRIPTION
**Pull request by szepeviktor**
- do not export development files
- check this in each commit

[_source_](https://github.com/szepeviktor/debian-server-tools/blob/9abca88bda2b3805a7be0c9a535966d775da8d5f/webserver/Continuous-integration-Continuous-delivery.yml#L195-L217)

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -69,3 +69,16 @@ jobs:
...
+      -
+        name: "Check exported files"
+        run: |
+            EXPECTED="LICENSE,README.md,composer.json,ecs.php"
+            CURRENT="$(
+                git archive HEAD \
+                    | tar --list \
+                    | paste --serial --delimiters=","
+            )"
+            echo "CURRENT =${CURRENT}"
```
Comment (GSadee) [inline .github/workflows/build.yml:82]:
```suggestion
            echo "CURRENT=${CURRENT}"
```

**Review by GSadee — APPROVED** [review]
I have some doubts if this check in every build is needed, but it seems ok

**Comment by GSadee** [comment]
Thank you, Viktor! :tada:

**Comment by szepeviktor** [comment]
Glad to contribute.

File: .github/workflows/build.yml (hunk 1 above)
Comment (szepeviktor) [reply .github/workflows/build.yml:82]: This would unalign current and expected file list.

Comment (szepeviktor) [reply .github/workflows/build.yml:82]:
```
CURRENT =LICENSE,README.md,composer.json,ecs.php
EXPECTED=LICENSE,README.md,composer.json,ecs.php
```

**Comment by szepeviktor** [comment]
> I have some doubts if this check in every build is needed, but it seems ok

@GSadee If you fall in love with doubtfully necessary checks, please browse my "byte-level care" repository with a total of 22 jobs in all workflows.
https://github.com/szepeviktor/byte-level-care/tree/master/.github/workflows

**Pull request merged**